frontends/precinct-scanner/src/app_root.tsx

```diff
@@ -357,16 +346,20 @@ export function AppRoot({
     void storeAppState();
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [
-    isPollsOpen,
     ballotCountWhenBallotBagLastReplaced,
     initializedFromStorage,
     isSoundMuted,
   ]);
 
-  const togglePollsOpen = useCallback(() => {
-    dispatchAppState({ type: 'togglePollsOpen' });
+  const updatePollsState = useCallback(async (newPollsState: PollsState) => {
+    dispatchAppState({ type: 'updatePollsState', pollsState: newPollsState });
+    await config.setPollsState(newPollsState);
```

Comment:
We should log whenever this updates. If you look at the table on page 253 of VVSG it gives some specific language about logging opening and closing of polls and also needing to log "The “zero totals” check conducted before opening
the polling place or counting a precinct centrally"

Reply:
We are lacking _a lot_ of logging on precinct scanner and I held off on some of it because I know we'll have to do a general pass later. I think it may be cleaner to do all that (polls state change, precinct state change (your comment above), report printed, saved to card) in a separate PR?

Reply:
Yeah there are definitely a lot of other areas where we are missing logging, I think its probably more efficient in general to just add things when we're in that code as its a little hard later sometimes to remember exactly what the error cases are or whats happening, and doing all of the logging later is a task no one is going to want to do. But we don't have to block on it